client/state/comments/actions.js

```diff
@@ -62,6 +63,11 @@ export const requestCommentsList = query => ( {
 	query,
 } );
 
+export const requestCommentsTree = query => ( {
```

Comment:
Perhaps `requestCommentsTreeForSite`. Let's also add jsDoc here since I wouldn't know what's required in `query`. Perhaps in the style of `requestPostComments` above?